content/200-concepts/100-components/01-prisma-schema/09-index-configuration.mdx

```diff
@@ -129,17 +139,103 @@ model Post {
 }
 ```
 
-## Upgrading from previous versions
+### New `type` argument (PostgreSQL)
+
+In version `3.6.0` and later, a new `type` argument is available in PostgreSQL on the `@@index` attribute.
+This allows you to use `Hash` as the index access method, instead of the default `BTree` access method. The `Hash` access method can be much faster for inserts, but only supports checking for equality, whereas the `BTree` access method supports all comparison operators.
```

Comment:
What does this actually do to the index? What is the effect on the queries? Why would someone want to do that?

Reply:
This looks like what you are asking, but I didn't check deeper than that
https://kb.objectrocket.com/postgresql/types-of-indexes-in-postgresql-769
<img width="1005" alt="Screen Shot 2021-11-29 at 16 07 15" src="https://user-images.githubusercontent.com/1328733/143892339-f5a37967-3526-4b63-adad-9a96d87e7d62.png">

Reply:
So if you define a `Hash` type as the index access method, it means the index data is stored in a format that is much faster to search and insert; using less disk space as a bonus. It also means only the operations `=` or `<>` (not equals) will use the index to access the data; other operations such as `>`, `<`et.al. will NOT use the index, but do a table scan instead and will be much slower compared to a `BTree` (the default). Also this kind of index type will not make `ORDER BY` operations any faster. If you know your use of this index is limited to these few operations, it makes sense to use due to much smaller space requirements and faster performance.

Reply:
Thanks, have added more detail